app/helpers.php

```diff
@@ -1,6 +1,34 @@
 <?php
 
 /**
+ * Build a CSV from the supplied data.
+ *
+ * @param  array  $data
+ * @return \League\Csv $csv
+ */
+function buildCSV($data)
```

Comment:
Nitpick: I think it's customary to have helper methods use `snake_case` in Laravel. Or could this be renamed to `csv` perhaps (kinda like the built-in `view()` or `json()` helpers)?